classifier_and_summarizer/summarizer/summarizer_output.py

```diff
@@ -0,0 +1,41 @@
+''' This script defines the object structure
+for the output(s) of the summarizer and provides
+methods to access the data as required'''
```

Comment:
The script is based roughly on the objects as defined in contents.py ( Contents and _Content) . Added methods for retrieving the contents in a dict format so they can be printed for debugging.